cmd/cloud/setup.go

```diff
@@ -213,13 +213,14 @@ func checkAPIKeys(astroClient astro.Client, coreClient astrocore.CoreClient, arg
 			if err != nil {
 				return false, err
 			}
+		}
 
-			// Switch context
-			err = context.Switch(domain)
-			if err != nil {
-				return false, err
-			}
+		// Switch context
+		err = context.Switch(domain)
```

Comment:
nit: I think the `Switch`function in itself also does the exact same thing as 
```
if !context.Exists(domain) {
			err := context.SetContext(domain)
			if err != nil {
				return false, err
			}
		}
```

so in theory, having the logic simply like this should give the same result:
```
        domain := "astronomer.io"
        err = context.Switch(domain)
        if err != nil {
                return false, nil
        }
```

Reply:
we only switch to astronomer.io if context doesn't exist because the user might be using astronomer-dev.io. We only want to switch to astronomer.io if the user has no context